fedimintd/src/fedimintd.rs

```diff
@@ -133,11 +132,6 @@ struct ServerOpts {
     #[arg(long, env = FM_BIND_METRICS_API_ENV)]
     bind_metrics_api: Option<SocketAddr>,
 
-    /// List of default meta values to use during config generation (format:
-    /// `key1=value1,key2=value,...`)
-    #[arg(long, env = FM_EXTRA_DKG_META_ENV, value_parser = parse_map, default_value="")]
-    extra_dkg_meta: BTreeMap<String, String>,
```

Comment:
Waiting for integrator feedback if this is still in use and needs an alternative after removal.